tmt/convert.py

```diff
@@ -8,15 +8,24 @@
 import re
 import subprocess
 from io import open
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple, Union
 
 import fmf.utils
+# Used only for typing
 from click import echo, style
 
+import tmt.export
 import tmt.utils
 from tmt.utils import ConvertError
 
 log = fmf.utils.Logging('tmt').logger
 
+# It is not possible to use TypedDict here, because all keys are unknown
+DataType = Dict[str, Any]
```

Comment:
I'd propose a different name, there is way too many "data" packages flying around. Can we find more specific "data", e.g. `NitrateDataType` or `TCMSDataType`? What kind of data it'd contain?

Reply:
Agree, renamed to `NitrateDataType`. Looks like it contains metadata which `Makefile` contained before in beakerlib tests